README.md

```diff
@@ -1 +1,42 @@
-# Mobile-Version
\ No newline at end of file
+# Mobile-Version
+- This is the second part of building my portfolio website
+- I will create sections within the website where i will include my portfolio projects.
+- This sections of the mobile website include:
+    1. THe toolbar section
+    2. Works section
+    3. The About myself
+
+## Built With
+
+- HTML/CSS linters
+
+
+
+👤 **Kelvin Ben**
+
+- GitHub: [@Kevo-hacker] (https://github.com/Kevo-hacker)
+- Twitter: [@kevoyout144] (https://twitter.com/kevoyout144)
+- LinkedIn: [Kelvin Ben] (https://www.linkedin.com/in/kelvin-ben-323043173/)
+
+👤 
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+- Thank you Microverse for the chance to learn through others
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+
```

Comment:
- [ ] Great job using the Microverse template here for the `README.md` and `LICENSE` 👌, but I noticed that this link is broken, to fix it you should rename the `LICENSE` file to the `MIT.md` file 👍.